### PR TITLE
NAS-118147: NAS-1181147: HTML refactoring & improvements, sorted Inpu…

### PR DESCRIPTION
**Pull request by AlexKarpov98**
refactored & Improved .html files

### CODE REVIEW

File: src/app/modules/truecommand/truecommand-button.component.html [hunk 1]
```diff
@@ -1,23 +1,42 @@
...
+    class="truecommand-icon"
+    [class]="tcStatus.status"
     svgIcon="ix:truecommand_logo_white"
     matBadgeSize="small"
-    [matBadge]="tcStatus.status === TrueCommandStatus.Connected ? 'check' : tcStatus.status === TrueCommandStatus.Failed ? 'priority_high' : ''"
+    [matBadge]="
+      tcStatus.status === TrueCommandStatus.Connected
+        ? 'check'
+        : tcStatus.status === TrueCommandStatus.Failed
+        ? 'priority_high'
```
Comment (undsoft) [inline src/app/modules/truecommand/truecommand-button.component.html:38]: One more level of indenation or better yet is to introduce a getter.

File: src/app/pages/reports-dashboard/reports-dashboard.component.html [hunk 2]
```diff
@@ -26,12 +26,26 @@
     <div>
       <mat-icon>warning</mat-icon>
     </div>
-    <div>{{ timeDiffWarning }} {{ 'NAS/browser time mismatch can lead to the reporting data display misbehavior.' | translate }}</div>
+    <div>
+      {{ timeDiffWarning }}
+
+      {{
+        'NAS/browser time mismatch can lead to the reporting data display misbehavior.'
+          | translate
```
Comment (undsoft) [inline src/app/pages/reports-dashboard/reports-dashboard.component.html:34]: Looks weird, better to keep as one line or introduce a class property.

File: src/app/pages/sharing/components/shares-dashboard/shares-dashboard.component.html [hunk 3]
```diff
@@ -1,42 +1,97 @@
...
+    #nfsTable
+    [ngClass]="getNfsOrder()"
+    [expandableTableState]="nfsExpandableState"
+    [disabled]="true"
+    [conf]="nfsTableConf"
+  >
     <ng-container
       headerCustomContent
-      *ngTemplateOutlet="nfsServiceStatus === ServiceStatus.Loading ? statusLoading : statusButton; context: { status: nfsServiceStatus, count: nfsHasItems }"
+      *ngTemplateOutlet="
```
Comment (undsoft) [inline src/app/pages/sharing/components/shares-dashboard/shares-dashboard.component.html:35]: Structural attributes should go first

File: src/app/pages/storage-old/volumes/delete-dataset-dialog/delete-dataset-dialog.component.html [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
+    mat-dialog-content
+    ix-auto
+    ix-auto-type="dialog"
+    ix-auto-identifier="title"
+    class="dialog-content"
+  >
     <div *ngIf="attachments.length" class="attachments">
-      {{ 'These services depend on dataset {name} and will be destroyed if the dataset is deleted:' | translate: { name: dataset.name } }}
+      {{
+        'These services depend on dataset {name} and will be destroyed if the dataset is deleted:'
```
Comment (undsoft) [inline src/app/pages/storage-old/volumes/delete-dataset-dialog/delete-dataset-dialog.component.html:16]: Same here and in next file.

File: src/app/pages/system/alert/alert.component.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
+<ng-template #pageHeader>
```
Comment (undsoft) [inline src/app/pages/system/alert/alert.component.html:1]: This file shouldn't exist.

**Review by undsoft — APPROVED** [review]
Nice work.

My only concern is with removal of markuplint rules for indentation. If these rules conflict with multiline attributes, I'd rather fix multiline attributes or even make them ugly for now.
It's better to have a working indentation linter.

File: .markuplintrc [hunk 6]
```diff
@@ -1,14 +1,12 @@
...
-   }
- },
- "excludeFiles": ["./src/index.html"]
+  "parser": {
+    ".html$": "markuplint-angular-parser"
+  },
+  "rules": {
+    "attr-equal-space-after": "never",
+    "attr-equal-space-before": "never"
+  },
```
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply .markuplintrc:8]: @undsoft  It didn't allow such things (for i.e placing button text on new line like below), but it really improves readability I think.

`
        <button
          mat-button
          type="submit"
          color="primary"
          [disabled]="form.invalid || isLoading"
        >
          {{ 'Save' | translate }}
        </button>
`

File: src/app/core/components/bulk-list-item/bulk-list-item.component.html [hunk 7]
```diff
@@ -1,11 +1,17 @@
...
+    fxLayout="row"
+    fxLayoutAlign="center center"
+  >
     <ng-container [ngSwitch]="item.state">
       <mat-icon
         *ngSwitchCase="State.Initial"
         class="initial"
         [matTooltip]="item.message || 'Pending' | translate"
-      >radio_button_checked</mat-icon>
+      >
```
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply src/app/core/components/bulk-list-item/bulk-list-item.component.html:12]: well yes, but harder to find opening-closing tag (at least for me - updated version is better)

File: src/app/modules/common/dialog/confirm-dialog/confirm-dialog.component.html [hunk 8]
```diff
@@ -1,13 +1,21 @@
...
+      color="primary"
+      ix-auto-type="checkbox"
+      ix-auto-identifier="{{ secondaryCheckBoxMsg | uppercase }}"
+      (change)="secondaryCheckBoxEvent()"
+    >
+      <span id="confirm-label">{{ secondaryCheckBoxMsg | translate }}</span>
+    </mat-checkbox>
     <ix-tooltip *ngIf="tooltip" [message]="tooltip"></ix-tooltip>
-    <br><br>
+    <br />
```
[the remark this answers is not in the document]
Comment (AlexKarpov98) [reply src/app/modules/common/dialog/confirm-dialog/confirm-dialog.component.html:17]: I think it's better to keep closing tag, for some general convention (like in React for example)

**Comment by AlexKarpov98** [comment]
@undsoft 
Updated & also returned:

    "indentation": {
      "value": 2,
      "severity": "error"
    }
    
    in markuplint

File: .markuplintrc (hunk 6 above)
[the remark this answers is not in the document]
Comment (undsoft) [reply .markuplintrc:12]: Just tested it, this code is working okay for me with having those rules present.

File: src/app/modules/common/dialog/confirm-dialog/confirm-dialog.component.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (undsoft) [reply src/app/modules/common/dialog/confirm-dialog/confirm-dialog.component.html:17]: Html5 is the general convention :)

**Pull request merged**